install/install.ps1

```diff
@@ -0,0 +1,65 @@
+# Enable a default -Verbose flag for debug output.
+[CmdletBinding()]
+Param()
+
+# Halt script execution at the first failed command.
+$script:ErrorActionPreference='Stop'
+
+# We don't currently have good cross-platform options for determining the latest release version, so we require
+# knowledge of the specific target version, which the user should set as an environment variable.
+$version = $Env:AZUREAUTH_VERSION
+if ($null -eq $version) {
+    # Write-Error is terminal with ErrorActionPreference='Stop', so we continue to hit the exit
+    # and set an exit code.
+    Write-Error 'No $AZUREAUTH_VERSION specified, unable to download a release' -ErrorAction:Continue
+    exit 1

```

Comment:
I think we want `return 1` here so that executing this doesn't end the callers entire PS session. In old style terminal it closes, in Windows Terminal (new) it leaves a dead session tab.

Reply:
Hmm... I don't think `return 1` actually sets the exit code, which is important. I'll have to look into this. Good catch.